### DISCUSSION TITLE
Adding keyboard navigation to charts

### DISCUSSION THREAD

**Discussion by orion-cengage** [carbon-design-system/carbon-charts#1743] [General]
I just wanted to see if there are plans to add keyboard navigation to the charts themselves. We are getting feedback from a 3rd party we use to evaluate the a11y of our applications and they're saying the expectation is for users who are limited to using a keyboard to be able to tab through the various data points and see the popover with the details, so they have an equivalent experience to a mouse user.

**Comment by theiliad** [comment]
Hi @orion-cengage. We went over this topic quite a few times with our accessibility expert. The recommendation was to not add that level of depth and leave it to
- Tabular representation
- aria-labels
- The application itself offering a text-based alternative to the chart's insights

The reason being that it's very difficult to create an accurate (depending on your data) & easy-to-follow path for keyboard navigation in charts, specially with some charts offering very complex visualizations

**Reply by orion-cengage** [reply]
Thank you very much! That is actually very closely aligned with what I tried to explain to them. I appreciate the quick response!
